web/cypress/fixtures/coo/logging/make-resources.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd ./cypress/fixtures/coo/logging/openshift
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)